fe/fe-core/src/main/java/org/apache/doris/datasource/es/MappingPhase.java

```diff
@@ -130,6 +130,10 @@ private static void resolveDocValuesFields(SearchContext searchContext, ObjectNo
                         if (docValue) {
                             docValueField = colName;
                         }
+                    } else if (innerTypeObject.has("ignore_above")) {
```

Comment:
we need to test "doc_values": true

Reply:
consider implict "ignore_above": 256

Reply:
1. The default value of `doc_values`  is `true`, its behavior is consistent with explicit set to `true`.
2. [keyword param](https://www.elastic.co/guide/en/elasticsearch/reference/current/keyword.html#keyword-params)
> [ignore_above](https://www.elastic.co/guide/en/elasticsearch/reference/current/ignore-above.html)
Do not index any string longer than this value. Defaults to 2147483647 so that all values would be accepted. Please however note that default dynamic mapping rules create a sub keyword field that overrides this default by setting ignore_above: 256. 

`ignore_above` default value is `2147483647`. 
The special case is the keyword field generated by dynamic mapping, whose `ignore_above` setting is `256` , which will be shown in mapping explicitly, and we have taken it into consideration.